node_darwin.go

```diff
@@ -26,24 +26,19 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) (err error) {
 	node.UID = stat.Uid
 	node.GID = stat.Gid
 
-	// TODO: cache uid lookup
-	if u, nil := user.LookupId(strconv.Itoa(int(stat.Uid))); err == nil {
+	if u, err := user.LookupId(strconv.Itoa(int(stat.Uid))); err == nil {
```

Comment:
I think this was a bug (`u, nil := ...`), the error would have been swallowed, right?